ports.mk

```diff
@@ -60,6 +60,9 @@ dist-generic: $(EXECUTABLE) $(PLUGINS)
 	mkdir -p ./dist-generic/scummvm/data
 	mkdir -p ./dist-generic/scummvm/doc
 	cp $(EXECUTABLE) ./dist-generic/scummvm
+ifeq ($(BACKEND), atari)
+	m68k-atari-mint-flags -S ./dist-generic/scummvm/$(EXECUTABLE)
```

Comment:
I have been thinking about this, too. However when I found out that I don't really need to repack or rename anything, I was like ok, maybe you will let me slip this in. ;) (but it's true it would be nice to supply `readme.txt` in the installation process, maybe even separate `data` into `data` and `themes`). Will keep this in mind for my next PR(s).